### PR TITLE
System test cleanup

### PR DESCRIPTION
**Pull request by edsantiago**
Misspellings, broken code, missing tests

Signed-off-by: Ed Santiago <santiago@redhat.com>
```release-note
None
```

### CODE REVIEW

File: test/system/200-pod.bats [hunk 1]
```diff
@@ -493,30 +492,24 @@ spec:
...
-            run_podman '?' rm -t 0 --all --force --ignore
-
-            while read path dev; do
-                if [[ "$path" == "$lofile" ]]; then
-                    losetup -d $dev
-                fi
-            done < <(losetup -l --noheadings --output BACK-FILE,NAME)
-            rm ${lofile}
-        fi
     fi
```
Comment (edsantiago) [inline test/system/200-pod.bats:498]: None of this code is run: `skip` is like `return`. Fortunately none of this (except the `losetup -d`) is even needed, and I moved it to before the `skip`, and it's much easier than this loop anyway.

File: test/system/200-pod.bats [hunk 2]
```diff
@@ -493,30 +492,24 @@ spec:
...
     echo bfq > /sys/block/$(basename ${LOOPDEVICE})/queue/scheduler
 
+    # FIXME: #15464: blkio-weight-device not working
     expected_limits="
 cpu.max         | 500000 100000
 memory.max      | 5242880
 memory.swap.max | 1068498944
+io.bfq.weight   | default 50
```
Comment (edsantiago) [inline test/system/200-pod.bats:506]: So, this was not actually being tested

**Review by vrothberg — COMMENTED** [review]
LGTM

**Comment by vrothberg** [comment]
@containers/podman-maintainers PTAL

**Pull request merged**